README.md

```diff
@@ -77,11 +77,11 @@ const totalPrice = 500; // Pass in the total price in smallest currency unit (ie
 const currency = 'USD';
 patch.orders.createOrder({ total_price: totalPrice, currency: currency });
 
-// Create order with the issuedTo field (optional)
+// Create order with the issued_to field (optional)
 const amount = 1_000_000; // Pass in the amount in unit specified
 const unit = 'g';
-const issuedTo = { email: 'issuee@companya.com', name: 'Olivia Jones' };
-patch.orders.createOrder({ amount: amount, unit: unit, issuedTo: issuedTo });
+const issued_to = { email: 'issuee@companya.com', name: 'Olivia Jones' };
+patch.orders.createOrder({ amount: amount, unit: unit, issued_to: issued_to });
```

Comment:
@holtbp looks like that got merged with this one. I'll modify the changelog to explain the breaking change.